Xamarin.Forms.Core/BindableLayout.cs

```diff
@@ -181,14 +181,14 @@ void CreateChildren()
 
 		View CreateItemView(object item, int index)
 		{
-			return CreateItemView(item, index, _itemTemplate ?? _itemTemplateSelector?.SelectTemplate(item, null));
+			return CreateItemView(item, index, _itemTemplate ?? _itemTemplateSelector);
 		}
 
 		View CreateItemView(object item, int index, DataTemplate dataTemplate)
 		{
 			if (dataTemplate != null)
 			{
-				var view = (View)dataTemplate.CreateContent();
+				var view = (View)dataTemplate.CreateContent(item, null);
```

Comment:
This is just one instance, but we can now call this method - regardless if it is a selector or a template.